damus/Models/Mentions.swift

```diff
@@ -495,14 +498,17 @@ func make_post_tags(post_blocks: [PostBlock], tags: [[String]], silent_mentions:
                 continue
             }
             
-            if let ind = find_tag_ref(type: ref.key, id: ref.ref_id, tags: tags) {
-                let mention = Mention(index: ind, type: mention_type, ref: ref)
+            if find_tag_ref(type: ref.key, id: ref.ref_id, tags: tags) != nil {
+                // Mention index is nil because positional mentions from NIP-07 is deprecated.
```

Comment:
Typo: comment should say NIP-08, not NIP-07. For this comment and same comment below.

Reply:
Fixed in 120d6bd049bda8ba6c363ef9391f88518375df3b